eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/publisher/impl/AlertEmailPublisher.java

```diff
@@ -215,7 +214,6 @@ public PublishmentType getPluginType() {
                 .name("Email")
                 .type(AlertEmailPublisher.class)
                 .description("Email alert publisher")
-                .field("subject")
```

Comment:
Yes. 

As we already define the email subject in the policy definition. it's a little duplicated. How do you think?